leaf-ui/rappid-extensions/ElementInspector.js

```diff
@@ -191,22 +192,27 @@ var ElementInspector = Backbone.View.extend({
         this.userConstraintsHTML = $("#new-user-constraints").last().clone();
 
         // Load initial value and node name
-        this.$('.cell-attrs-text').val(this.intention.nodeName);
-        this.$('#init-sat-value').val(satisfactionValuesDict[this.intention.getInitialSatValue()].name);
+        this.$('.cell-attrs-text').val(this.model.get('intention').get('nodeName'));
+        this.$('#init-sat-value').val(satisfactionValuesDict[this.model.get('intention').getInitialSatValue()].name);
+        
+        // Checks which function types are available based on initial satisfaction values
         this.checkInitialSatValue();
-
-        if (!cell.attr(".satvalue/value") && cell.attr(".funcvalue/text") != "NB"){
-            cell.attr(".satvalue/value", 'none');
-            cell.attr(".funcvalue/text", ' ');
+        
+        if (!this.model.attr(".satvalue/value") && this.model.attr(".funcvalue/text") != "NB"){
+            this.model.attr(".satvalue/value", 'none');
+            this.model.attr(".funcvalue/text", ' ');
         }
-
+        
         // Turn off repeating by default
         this.repeatOptionsDisplay = false;
         // TODO what does this do?
         this.setRepeatConstraintMode("TurnOff");
 
         // Load initial value for function type in the html select element
-        var functionType = this.intention.dynamicFunction.stringDynVis;
+        if (this.model.get('intention').get('evolvingFunction') != null ) {
+            var functionType = this.model.get('intention').get('evolvingFunction').get('type');
+        }
+        else {var functionType = null;}
```

Comment:
If you don't want to do anything when evolving function is null, you can just put all of the below code into the if (this.model.get('intention').get('evolvingFunction') != null ) and remove the else